### ISSUE TITLE
Semantically Markup Multiple Choice Answers

### ISSUE THREAD

**Issue by jpdevries** [Thinkful-Ed/quizful#35]
Multiple Choice Answers Should be radio buttons, not `<li>`s. You can make them look however you want.
https://github.com/jpdevries/css-only-radio-buttons/#demo

![](http://j4p.us/3k2M19041l3c/Screen%20Shot%202016-06-27%20at%204.17.55%20PM.png)

**Comment by jpdevries** [comment]
### Proof of Concept:

https://github.com/jpdevries/css-only-radio-buttons/#demo

### Form

User input **should always be wrapped in a form**. No exceptions. Practice safe input.

### Events

Do not listen to the click event!!!! List to change and submit events. Not all users click! Even though essentially every browser will treat the "enter key" as a click, there is absolutely nothing in the spec advising them to do so. They just have to because the internet would be broken otherwise. 

### Keyboard Accessible

Native HTML Input Components come with freebies like implicit accessibility. All you have to do is use them and the rest is automatic!

### Screen Reader Accessible

![](http://j4p.us/2o0t0O1e041i/Screen%20Shot%202016-06-28%20at%2012.47.50%20PM.png)

![](http://j4p.us/000i0t1v0n0K/Screen%20Shot%202016-06-28%20at%2012.47.30%20PM.png)
